src/components/ColorOptions/ColorOptions.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import './ColorOptions.styles.css';
+function NewNote({ changeColor }) {
+
+  const colorOptions = [
+    "red",
+    "orange",
+    "yellow",
+    "green",
+    "teal",
+    "blue",
+    "darkblue",
+    "purple",
+    "pink",
+    "brown",
+    "gray",
+  ];
```

Comment:
the constant values should be defined outside the component definition, the constant could be moved to the constants file